cmd/kubeadm/app/cmd/config.go

```diff
@@ -380,7 +380,11 @@ func RunConfigView(out io.Writer, client clientset.Interface) error {
 		return err
 	}
 	// No need to append \n as that already exists in the ConfigMap
-	fmt.Fprintf(out, "%s", cfgConfigMap.Data[constants.InitConfigurationConfigMapKey])
+	if value, ok := cfgConfigMap.Data[constants.InitConfigurationConfigMapKey]; ok {
```

Comment:
@neolit123 I'm not super familiar with golang idioms, but the key here is that even when retrieving with the wrong key there was no error (simply returned empty string). So, I would think that we can use the same conditional but based on length of the string returned, not the 'ok' status?

Reply:
@neolit123 thanks for the explanation!